README.md

```diff
@@ -59,11 +59,10 @@ $ bundle exec tapioca
 Commands:
   tapioca --version, -v      # show version
   tapioca dsl [constant...]  # generate RBIs for dynamic methods
-  tapioca generate [gem...]  # generate RBIs from gems
+  tapioca gem [gem...]       # generate RBIs from gems
```

Comment:
That's just a copypasta from the actual help docs. Will keep that in line with how it is now. However, we should add sub-command help docs in the future as well. Because right now, the experience of running `bin/tapioca gem --help` just running the command isn't great.

Reply:
https://github.com/Shopify/tapioca/blob/44fb9d1f226538cd0a5c7e41c0b46c9626cdb566/README.md?plain=1#L90